File: SRC - PERL/00 - Estruturas Sequenciais/exerProp/09.pl
#!/usr/bin/env perl
use v5.10;

use strict;
use warnings;

use utf8;
use open qw(:std :utf8);

# Faça um programa que receba o valor so salário mínimo e o valor do salário de um funcionário, calcule e mostre a quantidade de salários mínimos que esse funcionário ganha.

print "Digite o valor do salário mínimo >> ";
my $salario_minimo = <>;
chomp $salario_minimo;

print "Digite o valor do salário do funcionário >> ";
my $salario_funcionario = <>;
chomp $salario_funcionario;

my $total_salarios = $salario_funcionario / $salario_minimo;

printf "O funcionário ganha %d salário(s) mínimo(s) \n", $total_salarios;

File: startingPoint/02-EstruturaSequencial/Exercicios_Resolvidos/09.py
# 10 - Faça um programa que receba o valor so salário mínimo e o valor do salário de um funcionário, calcule e mostre a quantidade de salários
# mínimos que esse funcionário ganha.

sal_minimo, sal_funcionario = [float(x) for x in input("Digite o valor do salário mínimo e o valor do seu salário; ").split()]

quant_sal_minimo = sal_funcionario / sal_minimo
quant_sal_minimo = round(quant_sal_minimo, 2)

print("Você recebe {0} salário(s) minimo(s)".format(quant_sal_minimo))

      

